src/uwtools/config.py

```diff
@@ -104,23 +106,47 @@ def dump_file(self, output_path):
         with open(output_path, 'w', encoding="utf-8") as file_name:
             yaml.dump(self.data, file_name, sort_keys=False)
 
+    class Loader(yaml.SafeLoader): #pylint: disable=too-many-ancestors
+        '''Yaml loader to add INCLUDE tag for yaml file concatenation'''
+
+        def __init__(self, stream: IO) -> None:
+            try:
+                self._root = os.path.split(stream.name)[0]
+            except AttributeError:
+                self._root = os.path.curdir
+
+            super().__init__(stream)
+
+    def add_include(loader: Loader, node: yaml.Node) -> Any: #pylint: disable=no-self-argument
+        '''method to implement INCLUDE tag for yaml file concatenation'''
+        #pylint: disable=no-member
+        filename = os.path.abspath(os.path.join(loader._root, loader.construct_scalar(node)))
+        with open(filename, 'r', encoding="utf-8") as _file:
+            return yaml.load(_file, YAMLConfig.Loader)
+
+    yaml.add_constructor('!INCLUDE', add_include, Loader )
+
 
 class F90Config(Config):
 
     ''' Concrete class to handle Fortran namelist files. '''
 
-    def __init__(self, config_path):
+    def __init__(self, config_path=None, string=None):
 
         ''' Load the file and update the dictionary '''
+        self.string = string
         super().__init__(config_path)
-
         self.update(self._load())
 
     def _load(self):
         ''' Load the user-provided Fortran namelist path into a dict
         object. '''
-        with open(self.config_path, 'r', encoding="utf-8") as file_name:
-            cfg = f90nml.read(file_name)
+        if self.string is not None:
+            cfg = f90nml.reads(self.string)
+        else:
+            with open(self.config_path, 'r', encoding="utf-8") as file_name:
+                cfg = f90nml.read(file_name)
+        self.config_obj = cfg
```

Comment:
The changes to this class are outside the scope of the work, and it is unclear that it's needed. Since YAGNI applies here, please remove the modification.